src/api/index.ts

```diff
@@ -114,6 +115,11 @@ export default class ApiService {
 
   clearCurrentUser = () => {
     currentUser = undefined;
+    ApiService.clearCurrentUserFromStorage();
+  }
+
+  // because we need to call it from authContext which doesn't have access to ApiService
+  static clearCurrentUserFromStorage = () => {
```

Comment:
Is this static method still necessary after adding `forgetUser` to the auth util?

Reply:
Yes. `forgetUser` calls this function.
The problem is that auth/utils [not always](https://github.com/Coding-Coach/find-a-mentor/blob/45bff76e52632f4e81468fa00125b8f500217c89/src/utils/auth.js#L159-L160) has a reference to API so if it has reference it calls to `clearCurrentUser` but if not (it means that apiService is not initiated), it calls `clearCurrentUserFromStorage` directly.


We definitely should improve the design / structure of the auth / user area. We 2 providers and 2 services.
I did some research about auth0 react wrapper, it can help us clear our code but I didn't want to include this refactor in this PR to avoid big changes at once.

Reply:
https://github.com/Coding-Coach/find-a-mentor/pull/925/commits/5e14c29c1d6ca4dd701dff2be80c4ed4f5d4a0ef

Reply:
Sounds good.  Thanks.